geograypher/cameras/cameras.py

```diff
@@ -1244,7 +1244,7 @@ def calc_line_segments(
             range(len(self.cameras)), desc="Building line segments per camera"
         ):
             # Get the image filename
-            image_filename = str(self.get_image_filename(camera_ind, absolute=False))
+            image_filename = str(self.get_image_filename(camera_ind))
```

Comment:
This function is only used in `cameras.triangulate_detections`, so we aren't in danger of this change propagating errors. I don't know how this bug was introduced by me, but essentially the detector is storing images as absolute paths so the `absolute=False` is a problem.